build/npm/import-language-xlfs.js

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+let path = require("path");
+let gulp = require('gulp');
+let minimist = require('minimist');
+
+function update(options) {
+	let location = options._;
+
+	gulp.src(`${location}/drop/loc/**/*.xlf`)
+		.pipe(gulp.dest(function (file) {
+			let firstSlashIndex =  file.relative.indexOf('\\');
```

Comment:
Is this going to work cross-platform?

Reply:
I used the relative file path for this and assuming the drop zip files are the same one, it should work, I'll test it on my mac as soon as I can.